html5/js/MediaSourceUtil.js

```diff
@@ -27,14 +27,9 @@ const MediaSourceConstants = {
 
   CODEC_STRING: {
     "aac+mpeg4": 'audio/mp4; codecs="mp4a.40.2"',
-    //"aac+mpeg4"		: 'audio/mp4; codecs="aac51"',
```

Comment:
It took **a lot** of effort to figure out those values, so I am sad to see them go!
But this isn't used and anyway the git history still has them.